src/components/PopUps/PopUpMessageBox.tsx

```diff
@@ -113,6 +114,16 @@ const PopUpMessageBox = ({message}: PopUpMessageProps) => {
                 <Typography level={"body-sm"}>
                     {messageStr}
                 </Typography>
+                {button && (
+                    <Button
```

Comment:
Sorry i forgot to post this in the last batch of comments: can we right align this button?

Reply:
Sorry for the confusion. I meant in the UI, we can have something like
```
.pop-up-message-box-actions-container {
    display: flex;
    justify-content: flex-end;
}
```

```
<Box className="pop-up-message-box-actions-container">
    <Button/>
</Box>
```

Then it would look like
![image](https://github.com/user-attachments/assets/39d16e27-f9b7-46db-9311-ef95ab0d7a8f)